.github/workflows/nuget.yml

```diff
@@ -0,0 +1,30 @@
+name: Nuget Deploy
+
+concurrency: nuget_deploy
+
+on:
+  push:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup .NET Core @ Latest
+        uses: actions/setup-dotnet@v1
+        with:
+          source-url: https://nuget.pkg.github.com/EleventhHourGames/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{secrets.NUGET_PAT}}
```

Comment:
We probably need to add this secret to this project?
I borrowed this from `LastEpoch.Matchmaking.Shared` so we could grab the secret from there but I have no access to the project settings.